README.md

```diff
@@ -2,6 +2,37 @@
 
 [![Maintainability](https://api.codeclimate.com/v1/badges/36f6ce4580dafa42efc1/maintainability)](https://codeclimate.com/github/lendi-au/StackJanitor/maintainability)
 
-StackJanitor cleans up CloudFormation stacks based on **TTL** and **tags**.
+StackJanitor will monitor deployed development AWS CloudFormation Stacks and eliminate unused stacks based on the events activities to mitigate cloud costs.
```

Comment:
I'm not sure how I feel about the usage of the word "monitor" since this is all entirely event driven 🤔 but I'm also not sure what we could change it to. 
"track"?